### ISSUE TITLE
新浪新闻文本分类结果随机

### ISSUE THREAD

**Issue by liutianling** [KaiyuanGao/text_claasification#1]
我跑新浪新闻十分类的时候，结果直接随机，不知道怎么回事，请教一下？

INFO:tensorflow:***** Eval results *****
INFO:tensorflow:  eval_accuracy = 0.1
INFO:tensorflow:  eval_loss = 7.7761593
INFO:tensorflow:  global_step = 15625
INFO:tensorflow:  loss = 7.7761593

四块GPU好像也没有用起来，
 Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0     19900      C   python                                     10649MiB |
|    1     19900      C   python                                       215MiB |
|    2     19900      C   python                                       215MiB |
|    3     19900      C   python                                       215MiB |

我该怎么改动，谢谢。。。
